.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    parser: 'babel-eslint',
```

Comment:
not sure about this. the parser description says this:
> babel-eslint is a parser that allows ESLint to run on source code that is transformed by Babel.

but only the mandelbrot client-side theme code is transformed by Babel for us. i'm afraid if maybe this will allow new / experimental code that won't conform to our supported node versions.

Reply:
Indeed, I moved this to Mandelbrot repository specifically. I changed the EcmaScript version to 2017 as it seems to be the lowest we can go to support all the current syntax and still be compatible with Node 10.